app/Routes.hs

```diff
@@ -3,13 +3,41 @@ module Routes
 
 import Control.Monad (MonadPlus (mplus), msum)
 import Control.Monad.IO.Class (liftIO)
+import Controllers.Course (retrieveCourse, courses, courseInfo, depts)
```

Comment:
Looking ahead to other renaming, here I would use an aliased import (e.g. to `CoursesController`), so that you can use consistent naming conventions across controllers.

For example, `courses` can become `index` (and below for the graphs controller, `graphs` can become `index`).

Reply:
Just to clarify, do you mean change the import for graphs to something like this?
`import Controllers.Graph as GraphControllers`

and change the controller mappings for the `/graph` endpoint to ` ("graph", GraphControllers.graphResponse)`

Reply:
Yes exactly!